lib/sandbox.js

```diff
@@ -757,6 +757,66 @@ Sandbox.prototype.revokeToken = function (token, cb) {
     return cb ? promise.nodeify(cb) : promise;
 };
 
+/**
+ * List versions of a given node module that are available on the platform
+ * 
+ * @param {Object} options - Options
+ * @param {String} options.name - Name of the node module
+ * @param {Function} [cb] - Optional callback function for node-style callbacks
+ * @returns {Promise} A Promise that will be fulfilled with the token
+ */
+Sandbox.prototype.listNodeModuleVersions = function (options, cb) {
+    var request = Superagent
+        .get(this.url + `/api/env/node/modules/${encodeURIComponent(options.name)}`)
+        .set('Authorization', 'Bearer ' + this.token);
```

Comment:
It is not an authenticated operation. Would you prefer that I remove this header?